src/components/propositionalLogic/wiki.tsx

```diff
@@ -6,8 +6,31 @@ class WikiPropositionalLogic extends React.Component {
 
 	public render() {
 		return <div className={styles.site}>
-			<p>Hier soll ein kleines Wiki hin, ggf. mit infos zur Aussagenlogik,
-				quellen zu literatur oder eingabekonventionen.</p>
+			<p><h1>Allgemeines</h1>
```

Comment:
The <p> stands for paragraph. So it usually doesn't include the heading and it really is paragraph by paragraph.